sdk-base/src/main/java/com/mapbox/maps/plugin/ConfigProperties.kt

```diff
@@ -24,6 +24,22 @@ enum class ScrollMode(val value: String) {
   HORIZONTAL_AND_VERTICAL("horizontal-and-vertical"),
 }
 
+/**
+ * The enum controls how the puck is oriented
+ *
+ * @param value String value of this property
+ */
+enum class PuckBearingSource(val value: String) {
```

Comment:
Shoudn't this class also be generated?

Reply:
This is generated. We don't need to update anything in style-gen 😄